src/main/java/nl/novi/TechItEasy/dtos/TelevisionOutputDto.java

```diff
@@ -27,6 +30,9 @@ public class TelevisionOutputDto {
     private Integer originalStock;
     private Integer sold;
     public CIModuleOutputDto ciModule;
+    public RemoteControllerOutputDto remoteController;
```

Comment:
Als je in de TelevisionOutputDto een RemotecontrollerOutputDto zet, let er dan op dat je niet ook in RemotecontrollerOutputDto een TelevisionOutputDto zet. Als je dat doet krijg je namelijk last van recursie problemen (een oneindige loop)